src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue

```diff
@@ -188,12 +192,11 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-
 .forwarded-confirmation {
 	&__emptycontent {
-		width: 280px;
+		width: 100%;
 		text-align: center;
-		margin: 20px !important;
+		padding-top: 20px !important;
```

Comment:
this looks pretty broken when rebased on master.
Can you rebase and post screenshots?

Reply:
Ok